### PR TITLE
MM-2165 Added a Identifier to our testscipts resources

### PR DESCRIPTION
**Pull request by eduardderijcke**
@ArdonToonstra  Could you please review by adding the Identifier to our testscripts resources?

### CODE REVIEW

**Comment by ArdonToonstra** [comment]
Can you generate and run Touchstone testscripts too? 🗡️

File: src/BgLZ-3-0/Cert/_reference/resources/bglz-0-1-AllergieIntolerantie-01.xml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ArdonToonstra) [inline src/BgLZ-3-0/Cert/_reference/resources/bglz-0-1-AllergieIntolerantie-01.xml:1]: Not sure if this is a problem? It is sometimes added.. is it added by the script?

**Review by ArdonToonstra — COMMENTED** [review]
Perhaps also mentioned in PR or somewhere, that all fixtures now have the same indent and all schema's are removed.

File: src/BgLZ-3-0/Cert/_reference/resources/bglz-0-1-AllergieIntolerantie-01.xml (hunk 1 above)
Comment (eduardderijcke) [reply src/BgLZ-3-0/Cert/_reference/resources/bglz-0-1-AllergieIntolerantie-01.xml:1]: I've missed this addition that has been automatically added by the script, so I'll delete this.

**Comment by eduardderijcke** [comment]
PR gesloten ivm uitstel wijziginsverzoek naar oktober.

**Pull request closed without merging**